naga/tests/wgsl_errors.rs

```diff
@@ -241,15 +241,15 @@ fn constructor_parameter_type_mismatch() {
                 _ = mat2x2<f32>(array(0, 1), vec2(2, 3));
             }
         "#,
-        r#"error: automatic conversions cannot convert `array<{AbstractInt}, 2>` to `vec2<f32>`
+        "error: automatic conversions cannot convert `array<{AbstractInt}, 2>` to `vec2<f32>`
   ┌─ wgsl:3:21
   │
 3 │                 _ = mat2x2<f32>(array(0, 1), vec2(2, 3));
   │                     ^^^^^^^^^^^ ^^^^^^^^^^^ this expression has type array<{AbstractInt}, 2>
-  │                     │            
+  │                     │\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20
```

Comment:
This has nothing to do with the rest of the change, but I got tired of fighting with my editor to preserve the trailing whitespace in this file.